dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -238,7 +238,8 @@ private boolean problemFilter(Map.Entry<SymbolProblem, ClassFile> entry) {
    *     Virtual Machine Specification: 5.4.3.4. Interface Method Resolution</a>
    */
   @VisibleForTesting
-  Optional<SymbolProblem> findSymbolProblem(ClassFile classFile, MethodSymbol symbol) {
+  Optional<LinkageProblem> findLinkageProblem(
+      ClassFile classFile, MethodSymbol symbol, ClassFile sourceClassFile) {
```

Comment:
Is it possible for these three arguments to indicate more then one linkage problem?